src/content/en/2022/seo.md

```diff
@@ -1,22 +1,754 @@
 ---
 #See https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Authors'-Guide#metadata-to-add-at-the-top-of-your-chapters
 title: SEO
-#TODO - Review and update chapter description
 description: SEO chapter of the 2022 Web Almanac covering crawlability, indexability, page experience, on-page SEO, links, AMP, internationalization, and more.
 authors: [SophieBrannon, itamarblauer]
-reviewers: [mobeenali97, dwsmart, patrickstox, johnmurch, TusharPol]
-analysts: [derekperkins, csliva, jroakes]
+reviewers: [patrickstox, TusharPol, mobeenali97, dwsmart, johnmurch]
+analysts: [csliva, jroakes, derekperkins]
 editors: [MichaelLewittes]
 translators: []
 results: https://docs.google.com/spreadsheets/d/1qBQWxNKIAVJNOFwGlslT7AW0VAoK85Mf3nFvtw0QjVU/
-featured_quote: TODO
-featured_stat_1: TODO
-featured_stat_label_1: TODO
-featured_stat_2: TODO
-featured_stat_label_2: TODO
-featured_stat_3: TODO
-featured_stat_label_3: TODO
-unedited: true
+SophieBrannon_bio: TODO
+itamarblauer_bio: TODO
```

Comment:
@itamarblauer can you add a short bio here?

Reply:
@tunetheweb  we also need to add Mordy Oberstein + bio in the author section

Reply:
Added @mordy-oberstein as third author. Does the following order reflect the rough order of contribution (more -> less)? Or should it be reordered?:

```
authors: [SophieBrannon, itamarblauer, mordy-oberstein]
```

Same question about Reviewers and Analysts:

```
reviewers: [patrickstox, TusharPol, mobeenali97, dwsmart, johnmurch]
analysts: [csliva, jroakes, derekperkins]
```

Reply:
I think that's about right more or less, obviously the analysts are all the way through though. Not sure how last year's chapter ordered it? Might be worth referencing that.

Reply:
Bio for Mordy - 

Mordy Oberstein is the Head of SEO Branding at Wix. He also serves as a consultant for SEMrush and sits behind the mic of multiple SEO podcasts, including the SERP's Up podcast.

Reply:
@tunetheweb Bio for Itamar 

Itamar Blauer is an SEO expert based in London. He has a proven track-record of increasing rankings with SEO that is UX-focused, data-backed, and creative.

Reply:
> I think that's about right more or less, obviously the analysts are all the way through though. Not sure how last year's chapter ordered it? Might be worth referencing that.

Yeah it's always authors, reviewers, analysts, editors.
I more meant within those groups, but sounds like it;'s about right.

> Bio for Mordy -

Added

Reply:
> @tunetheweb Bio for Itamar
> 
> Itamar Blauer is an SEO expert based in London. He has a proven track-record of increasing rankings with SEO that is UX-focused, data-backed, and creative.

Added.